.github/workflows/issues-workflow.yml

```
@@ -0,0 +1,208 @@
+name: GitHub Issues Workflow Management
+
+on:
+  issues:
+    types: [opened, edited, labeled, assigned]
+  pull_request:
+    types: [opened, closed, merged]
```

actionlint (1.7.7)

invalid activity type "merged" for "pull_request" Webhook event. available types are "assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "edited", "enqueued", "labeled", "locked", "milestoned", "opened", "ready_for_review", "reopened", "review_request_removed", "review_requested", "synchronize", "unassigned", "unlabeled", "unlocked"

(events)